src/Sylius/Bundle/AdminBundle/Resources/views/Product/Attribute/attributeChoice.html.twig

```diff
@@ -1,6 +1,6 @@
 <div class="ui fluid action input" id="attributeChoice" data-action="{{ path('sylius_admin_render_attribute_forms') }}" style="margin-bottom: 15px;">
     {{ form_widget(form, {'attr': {'class': 'ui fluid search dropdown', 'id': 'sylius_product_attribute_choice'}}) }}
-    <button class="ui green labeled icon button">
+    <button id="addAttributesButton" class="ui green labeled icon button" data-title="{{ 'sylius.ui.no_new_attributes_selected'|trans }}">
```

Comment:
We should prefix these ids with ``sylius-`` = ``sylius-product-attributes-add-button`` or something like that?